### PR TITLE
feat: Evaluating resources

### PR DESCRIPTION
**Pull request by rtichauerv**
Added the functionality for users to evaluate a resource.

It lets users evaluate resources they hadn't evaluated before, and change the evaluations on resources they had evaluated.

### CODE REVIEW

File: spec/requests/resource_evaluations_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    before do
+      sign_in user
+    end
+
+    context 'when trying to create an existing evaluation' do
+      before do
+        create(:resource_evaluation, user:, resource:, evaluation: 4)
+      end
+
+      it 'updates the evaluation value' do
```
Comment (AlehandroL) [inline spec/requests/resource_evaluations_controller_spec.rb:25]: it 'doesn't update the evaluation value'

File: spec/requests/resource_evaluations_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        expect(ResourceEvaluation
+               .find_by(user_id: user.id, resource_id: resource.id)
+               .evaluation).to eq(3)
+      end
+
+      it "doesn't create a new object in the database" do
+        expect { perform }.not_to change(ResourceEvaluation, :count)
+      end
+
+      it 'shows the new evaluation value' do
```
Comment (AlehandroL) [inline spec/requests/resource_evaluations_controller_spec.rb:36]: shows the same/old evaluation value

File: spec/requests/resource_evaluations_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        perform
+        follow_redirect!
+        expect(response.body).to include('Your current evaluation is 3')
+      end
+    end
+
+    context 'when trying to create a new evaluation' do
+      it "doesn't display any evaluation for this user-resource before" do
```
Comment (AlehandroL) [inline spec/requests/resource_evaluations_controller_spec.rb]: doesn't display a **current** evaluation for this user-resource

File: app/views/resources/show.html.erb [hunk 4]
```diff
@@ -4,6 +4,9 @@
 
 <hr>
 
+<%= render partial: 'evaluation', locals: {resource_id: @service.id,
+  evaluation: @evaluation} %>
+
```
Comment (AlehandroL) [inline app/views/resources/show.html.erb:9]: please place the current average above or below the evaluation partial

**Review by AlehandroL — CHANGES_REQUESTED** [review]
minor changes in tests descriptions and partials positions

File: spec/requests/resource_evaluations_controller_spec.rb (hunk 1 above)
Comment (rtichauerv) [reply spec/requests/resource_evaluations_controller_spec.rb:25]: No, it tests that it does udpate it.
Before the test the evaluation was set in 4, the test changes it to 3.

File: spec/requests/resource_evaluations_controller_spec.rb (hunk 2 above)
Comment (rtichauerv) [reply spec/requests/resource_evaluations_controller_spec.rb:36]: No, it was set in 4 before the test and changed to 3

File: app/views/resources/show.html.erb (hunk 4 above)
Comment (rtichauerv) [reply app/views/resources/show.html.erb:9]: ok

**Review by AlehandroL — APPROVED** [review]

**Pull request merged**